src/Api/AdminConsole/Authorization/OrganizationRequirementHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿#nullable enable
+
+namespace Bit.Api.AdminConsole.Authorization;
+
+public static class OrganizationRequirementHelpers
```

Comment:
Minor non-blocker: for these helper classes/methods, maybe we should name them after the HTTP context rather than the Organization Requirement? I can see us using these helpers for things beyond the Organization Requirement.

Reply:
This has been rolled into the `HttpContextExtensions` class with the other extension methods.